src/FsPickler/FsPickler/FsPickler.fs

```diff
@@ -68,9 +80,10 @@ type FsPickler private () =
     /// <param name="value">Value to be cloned.</param>
     /// <param name="pickler">Pickler used for cloning. Defaults to auto-generated pickler.</param>
     /// <param name="streamingContext">Streaming context used for cloning. Defaults to null streaming context.</param>
-    static member Clone<'T> (value : 'T, [<O;D(null)>]?pickler : Pickler<'T>, [<O;D(null)>]?streamingContext : StreamingContext) : 'T =
-        let pickler = match pickler with None -> resolver().Resolve<'T> () | Some p -> p
-        let state = new CloneState(resolver(), ?streamingContext = streamingContext)
+    /// <param name="picklerResolver">Specify a custom pickler resolver/cache for serialization. Defaults to the singleton pickler cache.</param>
```

Comment:
Adding optional parameters to an existing public method is a runtime breaking change. Could this be a different API surface?